project.json

```diff
@@ -13,8 +14,14 @@
 		"Microsoft.AspNetCore.Server.Kestrel": "1.0.1",
 		"Microsoft.AspNetCore.StaticFiles": "1.0.0",
 		"Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
-		"Microsoft.Extensions.Logging.Console": "1.0.0"
-	},
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.AspNetCore.Mvc": "1.0.1",
+    "Microsoft.AspNetCore.Routing": "1.0.1",
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
+    "Microsoft.VisualStudio.Web.BrowserLink.Loader": "14.0.0",
+    "Microsoft.AspNetCore.Razor.Tools": "1.0.0-preview2-final",
+    "Microsoft.Extensions.Configuration.FileExtensions": "1.0.0",
+    "Microsoft.Extensions.Configuration.Json": "1.0.0"	},
```

Comment:
As previously, the indentation in this file is off.  This might have just been Visual Studio, it has a tendency to format this file wrong sometimes.  Either way, please switch it to tabs.